lib/embedded_ansible.rb

```diff
@@ -0,0 +1,24 @@
+class EmbeddedAnsible
+  APPLIANCE_ANSIBLE_DIRECTORY = "/opt/ansible-installer"
```

Comment:
Still not sure on this path.  We can always change later, so don't want it to hold up this PR, but just wanted to mention it.  cc @carbonin I was thinking /opt/awx_installer or /opt/ansible-tower-installer.  ansible-installer seems like a misnomer to me.